### PR TITLE
PMM-9642 PGSM Integration.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

File: pmm-tests/pg_stat_monitor_setup.sh [hunk 1]
```diff
@@ -63,33 +55,51 @@ echo "host    all             all              0.0.0.0/0                       m
 
 sleep 10
 service postgresql restart
-wget https://raw.githubusercontent.com/percona/pmm-qa/main/pmm-tests/tls-ssl-setup/postgres/init.sql
-su postgres bash -c 'psql -f init.sql'
+
+## NOTE
```
Comment (puneet0191) [inline pmm-tests/pg_stat_monitor_setup.sh:59]: I think we can remove these comments

File: pmm-tests/pg_stat_monitor_setup.sh [hunk 2]
```diff
@@ -63,33 +55,51 @@ echo "host    all             all              0.0.0.0/0                       m
...
+echo "CREATE USER pmm WITH PASSWORD 'pmm';" >> /home/postgres/init.sql
+echo "GRANT pg_monitor TO pmm;" >> /home/postgres/init.sql
+su postgres bash -c 'psql -f /home/postgres/init.sql'
+
+service postgresql start
+su postgres bash -c 'psql -c "CREATE DATABASE contrib_regression;"'
+su postgres bash -c 'psql -U postgres -c "CREATE EXTENSION pg_stat_monitor;"'
+
+## Running Queries
+wget https://raw.githubusercontent.com/percona/pmm-agent/main/testqueries/postgres/pg_stat_monitor_load.sql
```
Comment (puneet0191) [inline pmm-tests/pg_stat_monitor_setup.sh:105]: empty line here

File: pmm-tests/pg_stat_monitor_setup.sh [hunk 3]
```diff
@@ -63,33 +55,51 @@ echo "host    all             all              0.0.0.0/0                       m
...
-sleep 30
+
+echo "CREATE DATABASE sbtest1;" >> /home/postgres/init.sql
+echo "CREATE DATABASE sbtest2;" >> /home/postgres/init.sql
+echo "CREATE USER pmm WITH PASSWORD 'pmm';" >> /home/postgres/init.sql
+echo "GRANT pg_monitor TO pmm;" >> /home/postgres/init.sql
+su postgres bash -c 'psql -f /home/postgres/init.sql'
+
+service postgresql start
+su postgres bash -c 'psql -c "CREATE DATABASE contrib_regression;"'
```
Comment (puneet0191) [inline pmm-tests/pg_stat_monitor_setup.sh:101]: what about creating another database for pg_stat_statement and running similar load on that database

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: pmm-tests/pg_stat_monitor_setup.sh (hunk 3 above)
Comment (JiriCtvrtka) [reply pmm-tests/pg_stat_monitor_setup.sh:101]: Right I forgot that we discuseed it. I will fix it.

File: pmm-tests/pgsql_pgsm_setup.yml [hunk 4]
```diff
@@ -46,7 +46,6 @@
     with_items:
       - docker exec {{ pgsql_pgsm_container }} mkdir -p artifacts
       - docker cp ./pg_stat_monitor_setup.sh {{ pgsql_pgsm_container }}:/
-      - docker cp ./tls-ssl-setup/postgres/init.sql {{ pgsql_pgsm_container }}:/
```
Comment (JiriCtvrtka) [inline pmm-tests/pgsql_pgsm_setup.yml:49]: I moved it from file into sh script.

File: .gitignore [hunk 5]
```diff
@@ -10,3 +10,6 @@ pmm-tests/tls-ssl-setup/mysql/8*/*
 pmm-tests/tls-ssl-setup/mysql/setup*
 pmm-tests/tls-ssl-setup/mongodb/setup*
 pmm-tests/tls-ssl-setup/postgres/setup*
+pmm-tests/setup_pgsql_pgsm_12.log
```
Comment (puneet0191) [inline .gitignore:13]: Just keep it generic

**Review by puneet0191 — APPROVED** [review]
Thank you

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**